src/main/java/ru/portfolio/portfolio/parser/psb/DerivativeCashFlowTable.java

```diff
@@ -63,7 +63,7 @@ private boolean hasOpenContract() {
 
     private static AbstractMap.SimpleEntry<String, Integer> getCount(ExcelTable table, Row row) {
         String contract = table.getStringCellValue(row, CONTRACT);
-        int incomingCount = Math.abs(table.getIntCellValue(row, INCOUMING));
+        int incomingCount = Math.abs(table.getIntCellValue(row, ContractCountTableHeader.INCOMING));
```

Comment:
для единообразия без имени enum
```suggestion
        int incomingCount = Math.abs(table.getIntCellValue(row, INCOMING));
```